src/glslplugin/extensions/GLSLParameterInfoHandler.java

```diff
@@ -0,0 +1,121 @@
+package glslplugin.extensions;
+
+import com.intellij.codeInsight.lookup.LookupElement;
+import com.intellij.lang.parameterInfo.*;
+import com.intellij.psi.PsiElementResolveResult;
+import com.intellij.psi.PsiPolyVariantReference;
+import com.intellij.psi.PsiReference;
+import glslplugin.lang.elements.GLSLTokenTypes;
+import glslplugin.lang.elements.declarations.GLSLFunctionDeclaration;
+import glslplugin.lang.elements.declarations.GLSLFunctionDefinitionImpl;
+import glslplugin.lang.elements.declarations.GLSLParameterDeclaration;
+import glslplugin.lang.elements.expressions.GLSLExpression;
+import glslplugin.lang.elements.expressions.GLSLFunctionCallExpression;
+import glslplugin.lang.elements.statements.GLSLCompoundStatement;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * Created by abigail on 08/07/15.
+ */
+public class GLSLParameterInfoHandler implements ParameterInfoHandler<GLSLFunctionCallExpression, Object> {
+
+    @Override
+    public boolean couldShowInLookup() {
+        return true;
+    }
+
+    @Nullable
+    @Override
+    public Object[] getParametersForLookup(LookupElement item, ParameterInfoContext context) {
+        return new Object[0];
+    }
+
+    @Nullable
+    @Override
+    public Object[] getParametersForDocumentation(Object p, ParameterInfoContext context) {
+        return new Object[0];
```

Comment:
Maybe preallocate these two `new Object[0]`? Intellij already creates a lot of garbage. But not a big problem.